drizzle/0002_classy_war_machine.sql

```diff
@@ -133,7 +133,7 @@ select
     record_jsonb,
     old_record_jsonb;
 
-return coalesce(new, old);
+return null;
```

Comment:
what does this change do

Reply:
"After insert" trigger doesn't need to return anything, since the record was already written.